src/client/js/Panels/Crosscut/CrosscutController.js

```diff
@@ -226,6 +227,15 @@ define(['js/logger',
             crosscutsRegistry = GMEConcepts.getCrosscuts(memberListContainerID),
             len = crosscutsRegistry.length;
 
+        if (len === 0 && this._tabsHasBeenRequested === false) {
```

Comment:
As this would automatically update the model, I think this should not be the default behavior, rather some option controlled.